docs/website/static/community.html

```diff
@@ -227,6 +248,11 @@ <h2 class="divider line glow">Champions</h2>
                     <div class="card-text">
                       <div class="header">John Reese</div>
                       <div class="subheader">
+                        <a href="https://hachyderm.io/@jpreese" alt="Mastodon">
```

Comment:
Do you think we should check with John if he's OK with this getting included? (Or did you already?)

Reply:
It's a public profile just like his Twitter, but you're right, let's summon @jpreese to check.

Reply:
LGTM 🎉